server/data_updates/00029_20210315-144037_content_templates.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : Jérôme
+# Creation: 2021-03-15 14:40
+
+from superdesk.commands.data_updates import BaseDataUpdate
+from superdesk import get_resource_service
+
+
+class DataUpdate(BaseDataUpdate):
+
+    resource = "content_templates"
+    desks_field = "template_desks"
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        """Check templates to remove references to deleted desks (SDCP-488)"""
+        vocabularies_service = get_resource_service(self.resource)
+        desks_service = get_resource_service("desks")
+        templates_collection = mongodb_database[self.resource]
```

Comment:
this is already `mongodb_collection` argument

Reply:
oh right, I didn't even realised it was linked to resource, ok fixing it.